cp1404practicals/Prac_05/word_occurrences.py

```diff
@@ -0,0 +1,15 @@
+words_for_counting = {}
+
+number_of_words = input("Text: ")
```

Comment:
Not sure if this is a suitable variable name, kind of expect a int when I read "number of".